src/coreclr/gc/gcpriv.h

```diff
@@ -5122,6 +5193,8 @@ class gc_heap
     PER_HEAP_ISOLATED_FIELD_MAINTAINED size_t current_total_soh_stable_size;
     PER_HEAP_ISOLATED_FIELD_MAINTAINED uint64_t last_suspended_end_time;
     PER_HEAP_ISOLATED_FIELD_MAINTAINED uint64_t change_heap_count_time;
+    PER_HEAP_ISOLATED_FIELD_MAINTAINED uint64_t total_change_heap_count;
```

Comment:
Should these fields be `PER_HEAP_ISOLATED_FIELD_DIAG_ONLY` instead of `PER_HEAP_ISOLATED_FIELD_MAINTAINED`?

Reply:
yes I will change.

Reply:
yes! changed.